.github/workflows/pylint.yml

```diff
@@ -0,0 +1,31 @@
+name: Pylint
+
+on: 
+  pull_request:
+    branches: [ master ]
+    
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pylint
+        pip install -r requirments.txt
+    - name: Analysing the code with pylint
+      run: |
+        pylint `ls -R|grep .py$|xargs`
+    
+    - name: Test pytest 
+      run: |
+        pytest
+        
+    
```

Comment:
The new line at the end of the file should be blank.
I'll merge this, but fix it for next time.